lib/framework/framework.dart

```diff
@@ -251,3 +254,38 @@ class StatusItem {
 
   final CoreElement element;
 }
+
+class Toast extends CoreElement {
+  Toast({this.title, this.message}) : super('div', classes: 'toast') {
+    if (title != null) {
+      add(label(text: title));
+    }
+    add(div(text: message));
+  }
+
+  static const Duration animationDelay = Duration(milliseconds: 500);
+  static const Duration hideDelay = Duration(seconds: 4);
+
+  final String title;
+  @required
+  final String message;
+
+  void show() async {
+    await window.animationFrame;
+
+    element.style.left = '0px';
+
+    new Timer(animationDelay, () {
+      new Timer(hideDelay, _hide);
+    });
+  }
+
+  void _hide() {
+    element.style.left = '400px';
```

Comment:
is setting left to 400px needed? seems odd. I would hav expected toasts would animate from bellow not move 400px to the left.